src/status_im2/contexts/chat/home/view.cljs

```diff
@@ -102,33 +101,35 @@
 
 (defn home
   []
-  (let [selected-tab (reagent/atom :recent)]
-    (fn []
-      (let [pending-contact-requests (rf/sub [:activity-center/pending-contact-requests])]
-        [safe-area/consumer
-         (fn [{:keys [top]}]
-           [:<>
-            (if (= @selected-tab :contacts)
-              [contacts pending-contact-requests top]
-              [chats @selected-tab top])
-            [rn/view
-             {:style (style/blur-container top)}
-             [blur/view
-              {:blur-amount (if platform/ios? 20 10)
-               :blur-type   (if (colors/dark?) :dark (if platform/ios? :light :xlight))
-               :style       style/blur}]
-             [common.home/top-nav]
-             [common.home/title-column
-              {:label               (i18n/label :t/messages)
-               :handler             #(rf/dispatch [:show-bottom-sheet
-                                                   {:content home.sheet/new-chat-bottom-sheet}])
-               :accessibility-label :new-chat-button}]
-             [quo/discover-card
-              {:title       (i18n/label :t/invite-friends-to-status)
-               :description (i18n/label :t/share-invite-link)}]
-             [quo/tabs
-              {:style          style/tabs
-               :size           32
-               :on-change      #(reset! selected-tab %)
-               :default-active @selected-tab
-               :data           (get-tabs-data (pos? (count pending-contact-requests)))}]]])]))))
+  (fn []
+    (let [pending-contact-requests (rf/sub [:activity-center/pending-contact-requests])
+          selected-tab             (or (rf/sub [:messages-home-view/selected-tab])
```

Comment:
Suggestion for naming: Remove the `-view` ending, since I believe everybody can imply `:messages-home/selected-tab` would mean the Messages Home screen.

Reply:
Done, Thank you